zookeeper-server/src/main/java/org/apache/zookeeper/server/EphemeralType.java

```diff
@@ -188,24 +179,27 @@ public static EphemeralType get(long ephemeralOwner) {
     }
 
     /**
-     * Make sure the given server ID is compatible with the current extended ephemeral setting
+     * Make sure the given server ID is compatible with the current extended ephemeral setting.
      *
      * @param serverId Server ID
      * @throws RuntimeException extendedTypesEnabled is true but Server ID is too large
      */
     public static void validateServerId(long serverId) {
-        // TODO: in the future, serverId should be validated for all cases, not just the extendedEphemeralTypesEnabled case
+        // TODO: in the future, serverId should be validated for all cases,
```

Comment:
This is the kind of TODO that I'm wondering how the community tracks. In my perfect world, each is accompanied by a comment like "ZK-156".

Reply:
Support. This is also how akka community tracked TODOs and it was a good practice yet.